WcaOnRails/app/views/persons/_details.html.erb

```diff
@@ -0,0 +1,34 @@
+<div>
+  <div class="text-center">
+    <h2><%= @person.name %></h2>
+    <!-- Handle multiple sub ids. -->
+    <% if @previous_persons.any? %>
+      <h4>
+        (<%= t 'persons.show.previously' %> <%= @previous_persons.map { |person| "#{person.name} - #{person.country.name}" }.join(', ') %>)
+      </h4>
+    <% end %>
+    <% if @person&.user&.avatar %>
```

Comment:
Nice catch, even thought there would appear an exception at some point, I should've used an exclamation mark (`find_by_wca_id!`). And you're right that the `&` is not needed here =)